fut/core.py

```diff
@@ -139,6 +140,9 @@ def __init__(self, email, passwd, secret_answer, platform='pc', code=None, emula
             self.logger = logger(save=True)
         else:  # NullHandler
             self.logger = logger()
+        # Set initial delay
+        self.delayInterval = 1.3

```

Comment:
Should this default to `0` in order to not affect anybody's existing delay logic unless they explicitly enable the feature with `setRequestDelay`?